### DISCUSSION TITLE
Cursor on graphics tablet works wierdly

### DISCUSSION THREAD

**Discussion by FantomLis** [ppy/osu#21396] [Q&A]
### Type

Game behaviour

### Bug description

There are no problems on the mouse, but on the XP-Pen Star 06c graphics tablet, the cursor moves from the center of the screen to another screen (Video 1). However, in a normal game, you can see that the cursor clicks in several places at once (Video 2). (Sorry, translated by google)

P.S.: i fully reinstaller drivers, osu!lazer, this doen't work

### Screenshots or videos

https://user-images.githubusercontent.com/71078845/203611273-33831011-751d-4ae7-921e-f7fab3d08147.mp4

https://user-images.githubusercontent.com/71078845/203612227-fcc34a5b-330a-40cd-a107-c611a84e434d.mp4

### Version

2022.1123.0-lazer

### Logs

[database.log](https://github.com/ppy/osu/files/10077789/database.log)
[input.log](https://github.com/ppy/osu/files/10077791/input.log)
[network.log](https://github.com/ppy/osu/files/10077792/network.log)
[performance.log](https://github.com/ppy/osu/files/10077793/performance.log)
[runtime.log](https://github.com/ppy/osu/files/10077794/runtime.log)
[updater.log](https://github.com/ppy/osu/files/10077795/updater.log)

**Comment by bdach** [comment]
* High precision mouse on or off?
* In-game tablet handler on or off?
* Windows Ink on or off?
* Any custom drivers running in the background?

**Reply by FantomLis** [reply]
1 Off
2 On
3 On in pentablet driver
4 Valorant Vanguard (Anticheat working like a driver), Pentablet (Graphic tablet driver)

**Reply by bdach** [reply]
Does disabling windows ink help then?

**Reply by FantomLis** [reply]
No, it's not. Also when game is opened, others programs like Krita do the same bug, after closing osu, cursor starts work fine

p.s.; tablet fully works, this bug appears only when game is opened.

**Reply by bdach** [reply]
Does turning in-game tablet handler off help?

**Reply by FantomLis** [reply]
Nope, all the same

**Reply by FantomLis** [reply]
Driver settings (i dunno how to change language, so i tried my best to translate)
![image](https://user-images.githubusercontent.com/71078845/203830372-ef71e559-18f9-4660-a517-35d996e3ea8e.png)
Settings file (maybe this helps, i dunno (github don't allow to put ini files))
[settings.zip](https://github.com/ppy/osu/files/10086188/settings.zip)

**Reply by C9Glax** [reply]
In the first video you can see, that the cursor is also on the right monitor (although the driver should only map to the left one).
So _I think_ osu! is moving the cursor to a "wrong" position and the driver corrects to the "correct" one, thus causing the jumping.

**Reply by SosoG-null** [reply]
I'm getting the same issue with my Huion Kamvas Pro 13, expect it snaps to the egde of my screen instead of the centre.
https://user-images.githubusercontent.com/72688255/210136692-cd5cc50f-3682-4674-83c7-d94700a04179.mp4

**Reply by FantomLis** [reply]
And still silence...

**Reply by bdach** [reply]
I don't know how to help you. I don't own the hardware. I don't know what could be going wrong as the usual causes have been ruled out.
